src\backend\core\database\manager.py: error: Source file found twice under different module names: "backend.core.database.manager" and "src.backend.core.database.manager"
src\backend\core\database\manager.py: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules for more info
src\backend\core\database\manager.py: note: Common resolutions include: a) adding `__init__.py` somewhere, b) using `--explicit-package-bases` or adjusting MYPYPATH
